actors/cache/impl/zcache.go

```diff
@@ -217,3 +263,17 @@ func (m *ZCache) storeActorCode(shortAddress string, cid string) {
 	ctx := context.Background()
 	_ = m.shortCidMap.Set(ctx, shortAddress, cid, DummyTtl)
 }
+
+func (m *ZCache) storeEvmAddressMapping(f0 string, mapping *EvmAddressMapping) {
+	if f0 == "" {
+		m.logger.Sugar().Debugf("[storeEvmAddressMapping] - Trying to store mapping with empty f0 address")
+		return
+	}
+
+	ctx := context.Background()
+
+	err := m.evmAddressesMap.Set(ctx, f0, mapping, DummyTtl)
```

Comment:
This is not a DummyTtl anymore. Maybe we could rename it to "NotExpiringTtl" or something like this